lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -252,6 +254,126 @@ void FIRRTLTypesLowering::visitDecl(InstanceOp op) {
   opsToRemove.push_back(op);
 }
 
+void FIRRTLTypesLowering::visitDecl(MemOp op) {
+
+  auto type = op.getDataTypeOrNull();
+  // Bail out if the memory is empty
+  if (!type)
+    return;
+
+  SmallVector<FlatBundleFieldEntry, 8> fieldTypes;
+  flattenBundleTypes(type, "", false, fieldTypes);
+
+  // Mutable store of the types of the ports of a new memory
+  SmallVector<Type, 4> resultPortTypes;
+
+  DenseMap<StringRef, Value> newWires;
+  for (auto field : fieldTypes) {
+    // Create the new type for the memory. Copy every field, but
+    // update the "data" field to be the groundn type and the "mask"
```

Comment:
```suggestion
    // update the "data" field to be the ground type and the "mask"
```

Reply:
Thanks. Fixed in one of the fixup commits.